tests/installation/ipxe_install.pm

```diff
@@ -59,6 +59,16 @@ sub poweron_host {
     }
 }
 
+sub set_pxe_boot {
+    while (1) {
```

Comment:
no need for timeout in this loop? What if ipmi doesn't respond and `$stdout` never returns `m/Force PXE/`

Reply:
We don't have timeouts in the other functions as well. And, in case this is not working, we won't be able to use those machines anyway, and the global job timeout will hit the job. IMHO there is no need for additional timeout, as we won't block other jobs in this case (they would not be able to run due to the same reason).